test/cases/unspecified expansion.c

```diff
@@ -0,0 +1,6 @@
+#define EXPECTED_TOKENS 2*f(9)
```

Comment:
This is actually undefined behavior (in my C draft, 6.10.3.4, the example snippet) and it can expand either as `2*f(9)` or as `2*9*g`. It might make sense to find why clang and gcc do differently, but from the standard viewpoint this is fine. I added this test to set in stone the current preprocessor behavior.